app/src/androidTest/java/org/simple/clinic/overdue/AppointmentRepositoryAndroidTest.kt

```diff
@@ -1540,21 +1608,28 @@ class AppointmentRepositoryAndroidTest {
 
   data class Record(
```

Comment:
`Record` doesn't reveal intent.

Reply:
This is basically a convenience class used to tie together a `PatientProfile`, a `BloodPressureMeasurement`, and an `Appointment` for the purpose of testing overdue appointments.

I'll call it `RecordAppointment` instead since this is created for testing appointments, and the patient and blood pressure are just incidental properties that are necessary for creating appointments.

Reply:
This will also fit in well with the fact that it has a `toOverdueAppointment` method.